src/privileged/feature/api.js

```diff
@@ -0,0 +1,45 @@
+/* global ExtensionAPI */
+
+this.feature = class extends ExtensionAPI {
+
+  /**
+   * Extension Shutdown
+   * APIs that allocate any resources (e.g., adding elements to the browser’s user interface,
+   * setting up internal event listeners, etc.) must free these resources when the extension
+   * for which they are allocated is shut down.
+   */
+  onShutdown(shutdownReason) {
+    console.log("onShutdown", shutdownReason);
+    // TODO: remove any active ui
+  }
+
+  getAPI(context) {
+    const { studyUtils } = ChromeUtils.import(
+      context.extension.getURL("privileged/shieldUtils/jsm/StudyUtils.jsm"),
+    );
+    const { Feature } = ChromeUtils.import(
+      context.extension.getURL("privileged/feature/jsm/Feature.jsm"),
+    );
+
+    /*
+    console.log("config", config);
+    console.log("studyUtils", studyUtils);
+    console.log("Feature", Feature);
+    */
```

Comment:
Remove `console.log()`s? Not sure if `config` on L25 would be defined anyways since I don't see a `ChromeUtils.import()` for "privileged/Config.jsm".

Reply:
Will be removed. This is wip code atm.